model/src/main/proto/topic.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+import "thumbnail.proto";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Corresponds to the on-disk storage representing all available topics for play.
+message Classroom {
+  // Known topics that the player can play.
+  TopicSummary topic_summary = 1;
+
+  // The last time this classroom was updated.
+  int64 last_update_time_ms = 2;
+}
+
+// Corresponds to the list of topics that can be shown on the homescreen.
+message TopicList {
+  // Corresponds to the story promoted at the top of the homescreen. Either the story is in-progress, or it's a
+  // recommended story if no other stories are in progress.
```

Comment:
Extra white space nit before `recommended` from your last PR #175

Reply:
See my comment in #175. That extra space should only apply to TODOs.